src/orion/core/worker/experiment.py

```diff
@@ -130,10 +130,22 @@ def __init__(self, name, user=None):
                       name, user)
 
             if len(config) > 1:
-                version = max(map(lambda exp: exp['version'], config))
+                max_version = max(map(lambda exp: exp['version'], config))
+
+                if version is None:
+                    self.version = max_version
+                else:
+                    self.version = version
```

Comment:
This should be set at line 124, so this here would only be a if instead of if-else

Reply:
No, because if `version` is left to `None` and there is no configuration inside the database then `version` will never be set to `1`.

Reply:
oh ok!

Reply:
:upside_down_face: